adopt-a-dog/blog.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="zxx">
+<head>
+  <meta charset="utf-8">
+  <title>Adopt a Dog</title>
+</head>
+<body>
+    <header>
+        <div class="title">
+          <img src="adoptadog-logo.svg" alt="ADOPTADOG" class="logo">
+          <h1>Adopt A Dog</h1>
+          <h2>Seattle's premiere dog adoption agency</h2>
+        </div>
+        <div class="checkout">
+            <h6><!--shopping cart--> $1234.56</h6>
+            <h6>Checkout</h6>
+        </div>
+          <nav>
+            <ul class="nav">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="dogs.html">Dogs</a></li>
+                <li><a href="blog.html">Blog</a></li>
+            </ul>
+          </nav>
+      </header>
+         <article>
+        <h3> Traveling With Your Dog</h3>
+        <img src="../adopt-a-dog/images/patrick-hendry-221863-unsplash.jpg" alt="black dog and owner at a canyon">
+        <p> Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem
+            quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium
+            hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium
+            faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat.
+            Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, con </p>
+             <p>
+            et labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea ad
+            qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam
+            voluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit
+            ulparume nonseca estorer spernam. </p>
+
+            <h3>How To Walk Multiple Dogs</h3>
+            <img src="../adopt-a-dog/images/matt-nelson-259365-unsplash.jpg" alt="Group of dogs">
+
+            <p>
+            Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem
+            quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium
+            hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium
+            faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat.
+          </p>
+            <p>
```

Comment:
Make sure these p tags are in the same line as the content <p>l;dkfalkfjalj</p> not having the text on the next line. It might be a weird github thing, but check your editor and make sure it's all nested nicely.